src/main/java/edu/harvard/iq/dataverse/api/DatasetFieldServiceApi.java

```diff
@@ -168,7 +169,8 @@ public Response getByName(@PathParam("name") String name) {
                     .add("parentAllowsMultiples", parentAllowsMultiplesDisplay)
                     .add("solrFieldSearchable", solrFieldSearchable)
                     .add("solrFieldFacetable", solrFieldFacetable)
-                    .add("isRequired", isRequired));
+                    .add("isRequired", isRequired)
+                    .add("uri", uri));
```

Comment:
I'm happy to see someone updating this old API. 😄

Reply:
This is mostly a separate thing. Slava pointed out that this API didn't include the URI for a term (which was added with the OAI-ORE/Bag work from QDR). Since this was a one line change related to semantics, I added it here but the rest of the changes do not depend on this API having the extra field.